_data/speakers.yml

```diff
@@ -326,6 +332,7 @@ bios:
           </ol>
           This talk will propose a model for scaling as well as discussing
           possible ways Etsy will continue to scale as it grows.
+        video: LfqyhM1LeIU
```

Comment:
This is a link to `Bethany Macri - Retrospective: Scaling Infrastructure at Etsy`, not Michael's talk